Frameworks/CoreText/CTRun.mm

```diff
@@ -256,11 +261,15 @@ void CTRunDraw(CTRunRef run, CGContextRef ctx, CFRange textRange) {
         range.length = [string length];
     }
 
-    int numGlyphs = curRun->_characters.size();
+    int numGlyphs = [string length];
```

Comment:
I could not find a documentation to understand this so implemented the code as it was behaving before. I can just use _glyphAdvances here to be safe

---
In reply to: [77259915](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77259915) [](ancestors = 77259915)